src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/GATKDefaultCLPConfigurationArgumentCollection.java

```diff
@@ -0,0 +1,101 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import htsjdk.samtools.util.IOUtil;
+import htsjdk.samtools.util.Log;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.config.ConfigFactory;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Default arguments for GATK configuration.
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public final class GATKDefaultCLPConfigurationArgumentCollection extends CLPConfigurationArgumentCollection {
+    private static final long serialVersionUID = 1L;
+
+    public static final String TMP_DIR_NAME = "tmp-dir";
+    public static final String QUIET_NAME = "quiet";
+    public static final String USE_JDK_DEFLATER_LONG_NAME = "use-jdk-deflater";
+    public static final String USE_JDK_DEFLATER_SHORT_NAME = "jdk-deflater";
+    public static final String USE_JDK_INFLATER_LONG_NAME = "use-jdk-inflater";
+    public static final String USE_JDK_INFLATER_SHORT_NAME = "jdk-inflater";
+    public static final String NIO_MAX_REOPENS_LONG_NAME = "gcs-max-retries";
+    public static final String NIO_MAX_REOPENS_SHORT_NAME = "gcs-retries";
+
+    @Argument(fullName = TMP_DIR_NAME, doc = "List of temp directories to use.", common=true, optional=true)
+    public List<String> TMP_DIR = new ArrayList<>();
```

Comment:
While we're making these changes, could you camel case these variable names?  They've always been weirdly inconsistent with the rest of gatk.

Reply:
Done.